### PR TITLE
Add check-translations.sh - Tool to detect missing and useless strings

### PR DESCRIPTION
**Pull request by estellecomment**
See README.md added in PR for explanations.

### CODE REVIEW

**Comment by odelcroi** [comment]
Shall we keep the language we don't need? It brings a lot of unnecessary updates when merging the fork of element

**Comment by estellecomment** [comment]
I wonder if using the translation module will enable us to reuse the i18n tools from react-sdk, and this will be obsolete...

File: modules/tchap-translations/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Tchap Translations - a long story
+
+## What are all these translation files ?
+
+-   `tchap_translations_web.json` and `tchap_translations_react-sdk.json` : translations for strings that Tchap code adds to the web and react-sdk repos. Strings added in patch code should be translated in the react-sdk file. The rest in the web file.
```
Comment (odelcroi) [inline modules/tchap-translations/README.md]: I would merge those two files (maybe later in another PR) to simplify the maintenance

**Review by odelcroi — APPROVED** [review]

File: modules/tchap-translations/README.md (hunk 1 above)
Comment (estellecomment) [reply modules/tchap-translations/README.md]: done

**Comment by estellecomment** [comment]
CI passes locally.

**Pull request merged**